common/src/main/java/com/wynntils/screens/maps/SeaskipperMapScreen.java

```diff
@@ -0,0 +1,532 @@
+        List<SeaskipperPoi> filteredPois = new ArrayList<>();
+
+        for (int i = pois.size() - 1; i >= 0; i--) {
+            SeaskipperPoi poi = pois.get(i);
+            PoiLocation location = poi.getLocation();
+
+            if (location == null) continue;
+
+            float poiRenderX = MapRenderer.getRenderX(poi, mapCenterX, centerX, currentZoom);
+            float poiRenderZ = MapRenderer.getRenderZ(poi, mapCenterZ, centerZ, currentZoom);
+
+            float poiWidth = poi.getWidth(currentZoom, poiScale);
+            float poiHeight = poi.getHeight(currentZoom, poiScale);
+
+            BoundingBox filterBox = BoundingBox.centered(location.getX(), location.getZ(), poiWidth, poiHeight);
+            BoundingBox mouseBox = BoundingBox.centered(poiRenderX, poiRenderZ, poiWidth, poiHeight);
+
+            if (filterBox.intersects(textureBoundingBox)) {
+                filteredPois.add(poi);
+                if (hoveredPoi == null && mouseBox.contains(mouseX, mouseY)) {
+                    hoveredPoi = poi;
+                }
+            }
+        }
+
+        if (hoveredPoi != null) {
+            filteredPois.remove(hoveredPoi);
+            filteredPois.add(0, hoveredPoi);
+        }
+
+        return filteredPois;
+    }
+
+    private void renderHoveredSeaskipperDestination(PoseStack poseStack) {
+        if (hoveredPoi == null) return;
+
+        poseStack.pushPose();
+        poseStack.translate(width - SCREEN_SIDE_OFFSET - 250, SCREEN_SIDE_OFFSET + 40, 101);
+
+        final float centerHeight = 60;
+        final int textureWidth = Texture.TERRITORY_TOOLTIP_CENTER.width();
+
+        RenderUtils.drawTexturedRect(poseStack, Texture.TERRITORY_TOOLTIP_TOP, 0, 0);
+        RenderUtils.drawTexturedRect(
+                poseStack,
+                Texture.TERRITORY_TOOLTIP_CENTER.resource(),
+                0,
+                Texture.TERRITORY_TOOLTIP_TOP.height(),
+                textureWidth,
+                centerHeight,
+                textureWidth,
+                Texture.TERRITORY_TOOLTIP_CENTER.height());
+        RenderUtils.drawTexturedRect(
+                poseStack, Texture.TERRITORY_NAME_BOX, 0, Texture.TERRITORY_TOOLTIP_TOP.height() + centerHeight);
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                        CodedString.fromString("Level %d".formatted(hoveredPoi.getLevel())),
+                        10,
+                        10,
+                        CommonColors.ORANGE,
+                        HorizontalAlignment.LEFT,
+                        VerticalAlignment.TOP,
+                        TextShadow.OUTLINE);
+
+        float renderYOffset = 10;
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                        CodedString.fromString("Starting Coordinates: %d, %d"
```

Comment:
Does this make sense to display to the user? I think it is just for "internal use" to be able to show a destination selection box..? (Same goes for ending coordinates)

Reply:
Just another thing legacy had so brought it over but it's not really needed I don't think :)